candidates/tests/test_complex_fields.py

```diff
@@ -39,17 +40,20 @@ def setUp(self):
         self.person_extra = PersonExtra.objects.create(base=self.person, versions='[]')
         self.person_extra.update_complex_field(an_field, 'http://example.com/additional')
 
+    @skip("We're not auto adding fields to the form any more / yet")
```

Comment:
Yep. This is a feature that we're not using at all – previously the templates auto-generated a form based on DB fields. We've [moved to](https://github.com/DemocracyClub/yournextrepresentative/commits/master/elections/uk/templates/candidates/_person_form.html) making the form in the template manually, mainly so we can improve the UX.

The reason these tests are failing now is because the [_default_ templates are in the `election.uk` app now](https://github.com/DemocracyClub/yournextrepresentative/pull/307/commits/03858098eaa3158b954aeb155118a24d805b6bef#diff-97584e5a95ae547965f27982595fb5f4L8), so the generic template in `candidates/templates` isn't getting picked up any more.

This is an initial part of future moves away from the difference between the UK and generic functions.